templates/documentation/delivery-analytics/domain-referrer.md

```diff
@@ -1,8 +1,9 @@
 ---
-title: "Domain Referrer Restriction"
-excerpt: "Secure your videos and live streams by restricting from where the content is being consumed"
+title: Video Domain Restriction
+slug: domain-referrer
```

Comment:
`slug` can be removed, as confirmed by Doctave [here](https://api-video.slack.com/archives/C05DTNR3QDS/p1697618392282579)